projects/storefrontlib/src/cms-components/checkout/components/place-order/place-order.component.ts

```diff
@@ -9,27 +9,32 @@ import { CheckoutService, RoutingService } from '@spartacus/core';
 
 import { filter } from 'rxjs/operators';
 import { Subscription } from 'rxjs';
+import { FormBuilder, Validators } from '@angular/forms';
 
 @Component({
   selector: 'cx-place-order',
   templateUrl: './place-order.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class PlaceOrderComponent implements OnInit, OnDestroy {
```

Comment:
Breaking changes to document: 
- removed tAndCToggler and toggleTAndC
- renamed placeOrder to submitForm

Reply:
Done